relentless/ensemble.py

```diff
@@ -1,3 +1,21 @@
+"""
+Ensemble
+========
+
+This module defines a statistical mechanical thermodynamic ensemble. It is parametrized
+by a temperature, a particle type number, and either the volume or pressure of
+the system. For each pair in the ensemble, the radial distribution function (RDF)
+is also defined using an Akima spline.
```

Comment:
I think we should probably make this description a little more generic, since it is reproducing a lot of the `Ensemble` documentation. Maybe better would be to describe the intent of these objects more broadly (to define properties and statistics of simulated systems). For example, I think eventually we will add other types of distribution functions here, e.g., bond-length distribution.